base/applications/calc/resource.rc

```diff
@@ -11,16 +24,22 @@
 #include "resource.h"
 
 #ifndef IDC_STATIC
-#define IDC_STATIC -1
+#define IDC_STATIC    -1
+#endif
+
+#ifndef DS_SHELLFONT
+#define DS_SHELLFONT DS_SETFONT|DS_FIXEDSYS
```

Comment:
I'm wondering in which cases this DS_SHELLFONT is not defined...

Reply:
Well... most of these changes have been made in the far 2010, so I guess that it happened when I was working with MSVC6 and without a recent PSDK installed. Probably it can be removed actually.